native/cocos/audio/apple/AudioEngine-inl.mm

```diff
@@ -125,20 +125,25 @@ - (void)handleInterruption:(NSNotification *)notification {
             isAudioSessionInterrupted = false;
 
             NSError *error = nil;
-            [[AVAudioSession sharedInstance] setActive:YES error:&error];
+            if([AVAudioSession sharedInstance] respondsToSelector:@selector(setActive)){
```

Comment:
Why need to check it?

Reply:
[https://github.com/cocos/cocos-engine/issues/17004](url)

Reply:
[AVAudioSession sharedInstance] has been destroy

Reply:
The link above seems nothing with this issue.

Reply:
> Why need to check it?
[AVAudioSession sharedInstance] has been destroy, in same way, see the "issue two reproduce steps:"

Reply:
> The link above seems nothing with this issue.
the link has been deleted.

Reply:
> If so, why not just check [AVAudioSession sharedInstance] directly?
I will check it in this way.

Reply:
> If so, why not just check [AVAudioSession sharedInstance] directly?

good suggestion! I will check it in this way.

Reply:
> Why need to check it?

[AVAudioSession sharedInstance] has been destroy, in same way, see the "issue two reproduce steps:"

Reply:
> The link above seems nothing with this issue.

the link has been deleted.